server/test/unit/models/organization.js

```diff
@@ -18,6 +18,7 @@ describe('models.Organization', () => {
     assert(org.id);
     assert.deepStrictEqual(org.name, 'Kaiser Permanente');
     assert.deepStrictEqual(org.type, 'HEALTHCARE');
+    assert.equal(org.isMfaEnabled, false);
```

Comment:
I believe `equal` is deprecated in favor of `strictEqual` or `deepStrictEqual`. I tend to just use `deepStrictEqual` by default...